### DISCUSSION TITLE
Composer matching version error when installing

### DISCUSSION THREAD

**Discussion by shekenz** [vanilophp/framework#141] [Q&A] [answered]
Hi,

I get this error when I try to require vanilo/framework, while following the install instructions :

```
Could not find a matching version of package vite.config.js. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (dev).
```

I'm using Laravel v9.41.0 and PHP v8.1.7

How can I fix this ?

**Comment by fulopattila122** [comment]
The Vanilo Framework has no frontend assets. It has no dependencies or constraints on Vite or any other frontend tooling, thus your issue is unrelated to the Vanilo Framework

**Comment by shekenz** [comment]
I tested installing other packages and I have no issues though. It is weird because I understand how Vanilo have nothing to do with Vite, but it only happens when I try install this specific package.

Any idea where I should start digging ? Could it be some kind of compatibility issues with Laravel 9 ? They've been switching to Vite quiet recently...

**Reply by fulopattila122** [reply]
Try `composer why-not <package-name>`

**Reply by fulopattila122** [reply]
laravel/framework has neither dependencies on Vite, it's just the tool they started recommending in their app template laravel/laravel. So I would dig in the application setup

**Comment by shekenz** [comment, marked as answer]
Okay this is really silly, as always... I'm using zsh and in that shell, you have to put the version number in double quotes, otherwise it does some kind of interpretions. So to fix this :

```composer require vanilo/framework "^3.1"``` instead of ```composer require vanilo/framework ^3.1```

Now composer finally informs me that I'm missing some extensions, but that I know how to fix.

Thanks alot for the reactivity !
